src/main/scala/com/github/unchama/seichiassist/subsystems/subhome/bukkit/command/SubHomeCommand.scala

```diff
@@ -64,11 +66,33 @@ object SubHomeCommand {
   ]: SubHomeAPI: ConcurrentEffect: NonServerThreadContextShift: OnMinecraftServerThread](
     implicit scope: ChatInterceptionScope
   ): TabExecutor = BranchedExecutor(
-    Map("warp" -> warpExecutor, "set" -> setExecutor, "name" -> nameExecutor),
+    Map(
+      "warp" -> warpExecutor,
+      "set" -> setExecutor,
+      "name" -> nameExecutor,
+      "remove" -> removeExecutor
+    ),
     whenArgInsufficient = Some(printDescriptionExecutor),
     whenBranchNotFound = Some(printDescriptionExecutor)
   ).asNonBlockingTabExecutor()
 
+  private def removeExecutor[F[
+    _
+  ]: ConcurrentEffect: NonServerThreadContextShift: OnMinecraftServerThread: SubHomeWriteAPI] =
+    argsAndSenderConfiguredBuilder
+      .execution { context =>
+        val subHomeId = SubHomeId(context.args.parsed.head.asInstanceOf[Int])
+        val player = context.sender
+
+        val eff = for {
+          _ <- NonServerThreadContextShift[F].shift
+          _ <- SubHomeWriteAPI[F].remove(player.getUniqueId, subHomeId)
+        } yield MessageEffect(s"サブホームポイント${subHomeId}を削除しました。")
+
+        eff.toIO
+      }
```

Comment:
```suggestion
      .executionCSEffect { context =>
        val subHomeId = SubHomeId(context.args.parsed.head.asInstanceOf[Int])
        val player = context.sender

        Kleisli.liftF(
          SubHomeWriteAPI[F].remove(player.getUniqueId, subHomeId)
        ) >> MessageEffectF(s"サブホームポイント${subHomeId}を削除しました。")
      }
```

scalafmt通るかわかんないですが、こう書けるはず